python/jobs.py

```diff
@@ -56,6 +56,12 @@ def __init__(self):
         self.json_property(int, 'design_status')
         self.json_property(int, 'payment_status')
         self.json_property(int, 'shipping_status')
+        self.json_property(int, 'unread_notifications_count')
+        self.json_property(int, 'unread_job_info_notifications_count')
```

Comment:
This one does not appear to exist. One named `production`, does, though.

Reply:
I've added it in another PR inside private repo.

Reply:
hm, i haven't seen that one

Reply:
It's open now.